atr/get/sbom.py

```diff
@@ -183,41 +182,48 @@ def _outdated_tool_section(block: htm.Block, task_result: results.SBOMToolScore)
 
 
 def _conformance_section(block: htm.Block, task_result: results.SBOMToolScore) -> None:
+    block.h2["Conformance report"]
     warnings = [sbom.models.conformance.MissingAdapter.validate_python(json.loads(w)) for w in task_result.warnings]
     errors = [sbom.models.conformance.MissingAdapter.validate_python(json.loads(e)) for e in task_result.errors]
     if warnings:
-        block.h2["Warnings"]
+        block.h3[htm.icon("bi-exclamation-triangle-fill", ".me-2.text-warning"), "Warnings"]
```

Comment:
The `htm.icon` function adds the `bi-` prefix, so this will have `bi-bi-` as a prefix.